src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  background-color: aqua;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+
+  &__hands {
+    width: 400px;
+    height: 400px;
+    position: relative;
+
+    &-minutes {
```

Comment:
put &__hands-minutes separately
because otherwise, it is not possible to search by the class name

you should nest only element that are connected by sense but not just with the name prefix